shell/platform/fuchsia/flutter/flatland_external_view_embedder.cc

```diff
@@ -36,6 +36,20 @@ FlatlandExternalViewEmbedder::FlatlandExternalViewEmbedder(
   root_transform_id_ = flatland_->NextTransformId();
   flatland_->flatland()->CreateTransform(root_transform_id_);
   flatland_->flatland()->SetRootTransform(root_transform_id_);
+
+  if (intercept_all_input) {
+    input_interceptor_node_ = flatland_->NextTransformId();
+    flatland_->flatland()->CreateTransform(*input_interceptor_node_);
+
+    // Attach full-screen hit testing shield. Note that since the hit-region
+    // may be transformed (translated, rotated), we do not want to set
+    // width/height to FLT_MAX. This will cause a numeric overflow.
+    flatland_->flatland()->SetHitRegions(
+        *input_interceptor_node_,
+        {{{0, 0, kMaxHitRegionSize, kMaxHitRegionSize},
+          fuchsia::ui::composition::HitTestInteraction::
+              SEMANTICALLY_INVISIBLE}});
+  }
```

Comment:
Gfx is adding the child here, so lets just follow that? We should also add it to child_transforms_.

Reply:
Done